.github/workflows/release_charts.yaml

```diff
@@ -0,0 +1,29 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - rockcraft
+
+jobs:
+  release:
+    permissions:
```

Comment:
Not sure if this is necessary I think we got away without having this before, see [here](https://github.com/canonical/hostpath-provisioner/blob/master/.github/workflows/release-chart.yaml)

Reply:
ah -- didn't know.  I was just following 
https://helm.sh/docs/howto/chart_releaser_action/#github-actions-workflow